### ISSUE TITLE
Add Support for Pico S/C/Pro machines

### ISSUE THREAD

**Issue by hotzenklotz** [hotzenklotz/picobrew-server#8]
Right now, the picobrew-server only supports the Zymatic machine protocol. It would be cool to support newer machines from the `Pico` family as well.

- [ ] Document Protocol
- [ ] Implement Protocol

**Comment by hotzenklotz** [comment]
@chiefwigms According to the readme from https://github.com/chiefwigms/picobrew_pico you figured out the protocol for the Pico S/C/Pro, correct? I'd love to upstream these changes.

**Comment by hotzenklotz** [comment]
@chiefwigms Looks like the Pico [reads it's recipes as JSON](https://github.com/chiefwigms/picobrew_pico/blob/master/app/main/recipe_parser.py). 

- Is that something that you have done or is this official from Picobrew? 
- Do you mind sharing such a recipe as a reference? 
- Where can one download existing recipes from the Picobrew.com recipe builder?

**Comment by chiefwigms** [comment]
Yep - i have the Pico S/C/Pro protocol (loosely documented in routes_api.py)

The pico doesn't have a real recipe aspect - the server pushes down a string for each pack that lists each step.  I added a feature where you can create your own recipe and act as though it were one of Pico's Paks.  I used JSON for both session data & the recipe because it was nearly plug in for HighCharts (live session graphing and historical sessions) as well as creating custom recipes via Tabulator.

PicoBrew doesn't have a recipe builder for the Pico..

Here's what a recipe looks like (similar to the zymatic):
#Test/-1,-1,5.5,48,0,3,0,0,Preparing To Brew,110,0,0,2,Heating,110,7,0,1,Dough In,148,45,0,1,Mash 1,156,0,0,1,Mash 2,178,7,2,1,Mash Out,202,10,0,3,Hops 1,202,8,0,4,Hops 2,202,8,0,6,Hops 3,202,8,5,5,Hops 4,|2048 BYTE IMAGE|#

**Comment by hotzenklotz** [comment]
I see. I guess I will just make the Zymatic recipes available in the Pico string format then for now.

**Comment by dlgreenwald** [comment]
I think it may still be worth supporting Pico Pro api endpoints just for delivering the cleaning "recipes" as well as data logging for a manual brew even if the true custom recipes can't be supported.  If you are open to it I can look into what it would take and make a pull request if I'm successful.

**Comment by hotzenklotz** [comment]
@dlgreenwald Sounds good. Good for it. Happy to accept PRs.
